biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -818,6 +825,21 @@ subroutine UpdatePlantHydrLenVol(ccohort,site_hydr)
     real(r8) :: norm                         ! total root fraction used <1
     integer  :: nlevrhiz                     ! number of rhizosphere levels
 
+    ! added by Junyan May 29, 2020
+    real(r8) :: dbh                          ! the dbh of current cohort                                             [m]   
+    real(r8) :: dbh_0                        ! the dbh of the sappling at recuitment                                 [m]
+
+    real(r8) :: dbh_max                      ! the maximum dbh a PFT can have as observed                            [m]
+    real(r8) :: dbh_rev                      ! ratio, similar to RWC*   
```

Comment:
what is rwc*?

Reply:
rwc* is the relative water content, the way dbh_rev is calculated in the similar way as RWC* is calculated.